system/Cache/Handlers/PredisHandler.php

```diff
@@ -133,7 +133,7 @@ public function save(string $key, $value, int $ttl = 60)
             return false;
         }
 
-        if ($ttl) {
+        if ($ttl !== 0) {
```

Comment:
Should be positive-int.
```suggestion
        if ($ttl > 0) {
```